go/vt/vtadmin/cluster/cluster.go

```diff
@@ -388,6 +406,26 @@ func (c *Cluster) findWorkflows(ctx context.Context, keyspaces []string, opts Fi
 	}, nil
 }
 
+// GetGates returns the list of all VTGates in the cluster.
+func (c *Cluster) GetGates(ctx context.Context) ([]*vtadminpb.VTGate, error) {
+	// (TODO|@ajm188) Support tags in the vtadmin RPC request and pass them
+	// through here.
+	gates, err := c.Discovery.DiscoverVTGates(ctx, []string{})
+	if err != nil {
+		return nil, fmt.Errorf("DiscoverVTGates(cluster = %s): %w", c.ID, err)
+	}
+
+	// Not every discovery implementation will necessarily populate the Cluster
```

Comment:
This is pretty nitpicky so feel free to leave it, but a more straightforward phrasing might be:

```
// This overwrites any "Cluster" field populated by the discovery implementation
```

I know that the code is _right there_, but the current comment (to me) more readily implies that the "Cluster" field will be populated iff it doesn't already exist from the discovery implementation. Like I said... nitpicky. 😎

Reply:
No this distinction makes sense to me! Thank you!